Jenkinsfile

```diff
@@ -22,6 +22,11 @@ pipeline {
                 stage('Kotti') {
                     agent { label "aws-slave-m5-xlarge" }
                     steps {
+                        sh "curl -L -O https://go.dev/dl/go1.18.3.linux-amd64.tar.gz"
+                        sh "sudo rm -rf /usr/bin/go && sudo rm -rf /usr/local/go && sudo tar -C /usr/local -xzf go1.18.3.linux-amd64.tar.gz"
```

Comment:
`/usr/bin/go` is the default Go installation. `/usr/local/go` is the dir where we extract the downloaded Go artifacts to. Even though its normally redundant to remove the latter, it is useful when the instance gets reused and straggling PATH or artifacts otherwise may exist. Jenkins maintains the EC2 instances for a short period following a run to save on restart-startup cost/time, so rapid pushes can be run on the same "machine." 

The following stanzas specify the expected Go executable artifacts precisely to provide better information for the reader.